packages/ui/src/domainModules/collectionMammals/components/MammalManager/index.js

```diff
@@ -375,7 +375,18 @@ class MammalManager extends Component {
         break
       }
       case DEL_SUCCESS: {
-        setTimeout(() => this.handleSearchSpecimens())
+        const { currentTableRowNumber, isTableView } = this.props
+
+        this.handleSearchSpecimens(undefined, { openTableView: false }).then(
+          () => {
+            if (!isTableView) {
+              setTimeout(() => {
+                this.handleOpenTableView()
+                this.handleSetCurrentTableRowNumber(null, currentTableRowNumber)
```

Comment:
when I delete the last specimen in the table the row number becomes wrong
![screenshot 2019-02-05 10 57 44](https://user-images.githubusercontent.com/12738484/52265814-2f97e500-2935-11e9-96d7-67ad14c1ae62.png)

(if it's any help, this works in the resourceManager)